test/native/setup.js

```diff
@@ -168,21 +169,7 @@ jest.mock( 'react-native-safe-area', () => {
 	};
 } );
 
-// To be replaced with built in mocks when we upgrade to the latest version
-jest.mock( 'react-native-safe-area-context', () => {
-	const inset = { top: 0, right: 0, bottom: 0, left: 0 };
-	const frame = { x: 0, y: 0, width: 0, height: 0 };
-	return {
-		SafeAreaProvider: jest
-			.fn()
-			.mockImplementation( ( { children } ) => children ),
-		SafeAreaConsumer: jest
-			.fn()
-			.mockImplementation( ( { children } ) => children( inset ) ),
-		useSafeAreaInsets: jest.fn().mockImplementation( () => inset ),
-		useSafeAreaFrame: jest.fn().mockImplementation( () => frame ),
-	};
-} );
+jest.mock( 'react-native-safe-area-context', () => mockSafeAreaContext );
```

Comment:
Now that we use a newer version of `react-native-safe-area-context`, we can use the mock provided by the package ([reference](https://github.com/th3rdwave/react-native-safe-area-context#testing)).